GG/src/Font.cpp

```diff
@@ -527,9 +722,226 @@ Font::FormattingTag::FormattingTag(bool close) :
     close_tag(close)
 {}
 
+void Font::FormattingTag::Bind(const std::string& whole_text)
+{
+    TextElement::Bind(whole_text);
+    tag_name.Bind(whole_text);
+    for (std::vector<Substring>::iterator it = params.begin();
+         it != params.end(); ++it)
+    {
+        it->Bind(whole_text);
+    }
+}
+
 Font::FormattingTag::TextElementType Font::FormattingTag::Type() const
 { return close_tag ? CLOSE_TAG : OPEN_TAG; }
 
+bool Font::FormattingTag::operator==(const TextElement &rhs) const
+{
+    Font::FormattingTag const* ft = dynamic_cast<Font::FormattingTag const*>(&rhs);
+    return (ft
```

Comment:
not required, but I'd prefer the && at the end of lines here... just me being arbitrary, though...